### ISSUE TITLE
pip install asciitable fails for 0.7.0

### ISSUE THREAD

**Issue by cdeil** [taldcroft/asciitable#13]
$ sudo pip install asciitable
Password:
Downloading/unpacking asciitable
  Downloading asciitable-0.7.0.zip (306Kb): 306Kb downloaded
  Running setup.py egg_info for package asciitable

```
file asciitable.py (for module asciitable) not found
```

Installing collected packages: asciitable
  Running setup.py install for asciitable
    file asciitable.py (for module asciitable) not found
    file asciitable.py (for module asciitable) not found
    warning: install_lib: 'build/lib' does not exist -- no Python modules to install

```
file asciitable.py (for module asciitable) not found
file asciitable.py (for module asciitable) not found
```

Successfully installed asciitable
Cleaning up...

**Comment by taldcroft** [comment]
Ugh, forgot to change setup.py to reflect new package structure.  Fixed in 0.7.0.1 now, please try again.

**Comment by cdeil** [comment]
Works now. Thanks!
